<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>frdbsettings</class>
 <widget class="QDialog" name="frdbsettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Servidor</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Servidor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Banco</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Usu√°rio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Character set</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="leServer" />
       </item>
       <item>
        <widget class="QLineEdit" name="leDatabase" />
       </item>
       <item>
        <widget class="QLineEdit" name="leUser" />
       </item>
       <item>
        <widget class="QLineEdit" name="lePassword" >
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbCharset" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/agt_action_success.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="../res/rs.qrc" >:/png/buttons/16/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>leServer</tabstop>
  <tabstop>leDatabase</tabstop>
  <tabstop>leUser</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>cbCharset</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="../res/rs.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>frdbsettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>frdbsettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
